mocap_optitrack_driver/NatNetSDK/.gitignore

```diff
@@ -0,0 +1,3 @@
+include
+lib
+samples
```

Comment:
Should we keep only one gitignore or each in individual folder? 

If you want only one gitignore, 
[This](https://github.com/MOCAP4ROS2-Project/mocap4ros2_optitrack/blob/main/.gitignore) gitignore will do the same functionality as this one. Just change the name of main folder as it has been changed globally in the repo.

Reply:
Ok, lets adjust in in next Prs